lib/object_3d.dart

```diff
@@ -45,6 +53,29 @@ class _Object3DState extends State<Object3D> {
       // Load the object from a string
       _parseObj(widget.object!);
     }
+
+    final double maxSpeed = widget.maxSpeed.abs();
+    final double dampCoef = widget.dampCoef.clamp(0.001, 0.999);
+
+    _updateTimer = Timer.periodic(Duration(milliseconds: 16), (_) {
+      if (!mounted) return;
+      setState(() {
+        final double adx = _deltaX.abs();
+        final double ady = _deltaY.abs();
+        final double sx = _deltaX < 0 ? -1 : 1;
+        final double sy = _deltaY < 0 ? -1 : 1;
```

Comment:
Al these types are not necessary. These are inferred when using `final adx = _deltaX.abs();` for example

Reply:
I'm not a fan of omitting types. They cause more issues than the short-term convenience addresses. If you want them gone I can do that for you.

Reply:
It would be more consistent with the already existing code of this library, but whatever. I'm not going to block this merge because of this.

Reply:
done.